src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -64,10 +64,14 @@ public sealed override ImmutableArray<Symbol> GetMembers()
             EnsureAllMembersLoaded();
 
             var memberTypes = GetMemberTypesPrivate();
-            var builder = ArrayBuilder<Symbol>.GetInstance(memberTypes.Length + lazyNamespaces.Count);
+
+            if (_lazyNamespaces.Count == 0)
```

Comment:
The change makes sense to me. I dont know if compiler has a different code style for protected field names though